src/conn/conn_capacity.c

```diff
@@ -0,0 +1,504 @@
+/*
+ * Copyright (c) 2014-2018 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+#define	WT_CAPACITY_MIN_THRESHOLD	(10 * WT_MEGABYTE)
+#define	WT_CAPACITY_PCT			10
+/*
+ * If we're being asked to sleep a short amount of time, ignore it.
+ * A non-zero value means there may be a temporary violation of the
+ * capacity limitation, but one that would even out. That is, possibly
+ * fewer sleeps with the risk of more choppy behavior as this number
+ * is larger.
+ */
+#define	WT_CAPACITY_SLEEP_CUTOFF_US	100
+
+/*
+ * When given a total capacity, divide it up for each subsystem. We allow
+ * and expect the sum of the subsystems to exceed 100.
+ * We aim for:
+ *    checkpoint: 10% of total
```

Comment:
Checkpoint comment needs update (10% -> 5%), or just refer to defines below.

Reply:
Since I just changed it (incorrectly) today, I agree with just referring to the values below.